.circleci/config.yml

```diff
@@ -312,6 +325,16 @@ workflows:
 #          requires:
 #            - hold_production
 
+  smoke-test:
+    triggers:
+      - schedule:
+          cron: "30 8 * * *"
```

Comment:
what happens if it fails? I am sure it is graceful bit would there be any point running it sat and sun if no one is going to look or be able to respond?

Reply:
At some point I'll set up a new slack channel and implement slack orb notifications 👍

Reply:
And I've added that now...